src/libsystemd/sd-journal/journal-file.c

```diff
@@ -208,6 +208,7 @@ static bool journal_file_set_offline_try_restart(JournalFile *f) {
  * context without involving another thread.
  */
 int journal_file_set_offline(JournalFile *f, bool wait) {
+        int target_state;
```

Comment:
one could argue the appropriate type for this would be uint8_t, given that that's what we store in the header, but I guess that doesn't really matter...